corehq/util/context_processors.py

```diff
@@ -42,8 +42,13 @@ def get_per_domain_context(project, request=None):
     if hasattr(request, 'couch_user') and request.couch_user and project:
         allow_report_an_issue = request.couch_user.has_permission(project.name, 'report_an_issue')
     elif settings.ENTERPRISE_MODE:
-        allow_report_an_issue = (hasattr(request, 'couch_user') and request.couch_user and
-                                 request.couch_user.is_web_user())
+        if not(hasattr(request, 'couch_user') and request.couch_user):
```

Comment:
This could be simplified to 
```
if not getattr(request, 'couch_user'):
```

Reply:
Yeah good suggestion. Watch out though because the way you put it it'll raise an exception if that attr doesn't exist; unlike dict.get (where `None` is the default default), you need to pass `None` if you want that to be the default value. I'll change these two lines